src/main/java/karuna/karuna_backend/content/infrastructure/ContentController.java

```diff
@@ -31,4 +34,14 @@ ContentDTO getContentByPage(@PathVariable String page) {
         return contentService.getContentByPage(page);
     }
 
+    @Operation(summary = "Mass update content", description = "Getting json (REQUEST BODY) and updating all content by list")
+    @ApiResponse(
+            responseCode = "200",
+            description = "Updated contents",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = MassContentWrapper.class)))
+    @PutMapping
```

Comment:
Add name to endpoints like, @PutMapping(/name). Add 2 error codes to swagger doccumentaton I believe that would be 401 error code when jwt authorization fails and 403 if Sping security check for admin role fails. In the future we might think if we should merge both 401 and 403 errors into one 403 error which is Spring security default one, but that later.